composer.json

```diff
@@ -13,8 +13,8 @@
     "require": {
         "php": "^5.4|7.*",
         "puli/composer-plugin": "~1.0.0-beta8",
-        "puli/discovery": "~1.0.0-beta8",
-        "puli/cli": "~1.0.0-beta9"
+        "puli/discovery": "1.0.0-beta8",
```

Comment:
master is currently broken. we should be more flexible again when its fixed, as this will be a huge PITA to install if more than one repository depends on puli.